tests/test_merge_mono.py

```diff
@@ -0,0 +1,90 @@
+import json
+
+import pytest
+from fixtures import DataDir
+
+from pipeline.common.downloads import read_lines
+
+corpus_sample = """CORPUS 1
+CORPUS 2
+CORPUS 3
+CORPUS_SHARED 1
+CORPUS_SHARED 2
+CORPUS 4
+CORPUS 5
+"""
+
+news_2014_sample = """NEWS_2014 1
+NEWS_2014 2
+MONO_SHARED 1
+MONO_SHARED 2
+NEWS_2014 3
+CORPUS_SHARED 1
+NEWS_2014 4
+"""
+
+nllb_sample = """NLLB 1
+NLLB 2
+MONO_SHARED 1
+MONO_SHARED 2
+NLLB 3
+CORPUS_SHARED 2
+NLLB 4
+"""
+
+
+@pytest.mark.parametrize("task", ["src-en", "trg-ru"])
+def test_merge_mono(task: str):
```

Comment:
Does this test cover all the use cases? Would it be easier to run a bunch of regular tests instead of one run_task test to test the use cases separately?